bundle.yaml

```diff
@@ -1,33 +1,33 @@
+  istio-ingressgateway:      { charm: istio-ingressgateway,    channel: edge, scale: 1 }
+  istio-pilot:               { charm: istio-pilot,             channel: edge, scale: 1, options: { default-gateway: "kubeflow-gateway" } }
+  jupyter-controller:        { charm: jupyter-controller,      channel: edge, scale: 1 }
+  jupyter-ui:                { charm: jupyter-ui,              channel: edge, scale: 1 }
+  katib-controller:          { charm: katib-controller,        channel: edge, scale: 1 }
   katib-db:                  { charm: cs:~charmed-osm/mariadb-k8s-35, scale: 1, options: { database: katib } }
-  katib-db-manager:          { charm: cs:katib-db-manager-4,          scale: 1 }
-  katib-ui:                  { charm: cs:katib-ui-30,                 scale: 1 }
-  kfp-api:                   { charm: cs:kfp-api-12,                  scale: 1 }
+  katib-db-manager:          { charm: katib-db-manager,        channel: edge, scale: 1 }
+  katib-ui:                  { charm: katib-ui,                channel: edge, scale: 1 }
+  kfp-api:                   { charm: kfp-api,                 channel: edge, scale: 1 }
   kfp-db:                    { charm: cs:~charmed-osm/mariadb-k8s-35, scale: 1, options: { database: mlpipeline } }
-  kfp-persistence:           { charm: cs:kfp-persistence-9,           scale: 1 }
-  kfp-schedwf:               { charm: cs:kfp-schedwf-9,               scale: 1 }
-  kfp-ui:                    { charm: cs:kfp-ui-12,                   scale: 1 }
-  kfp-viewer:                { charm: cs:kfp-viewer-9,                scale: 1 }
-  kfp-viz:                   { charm: cs:kfp-viz-8,                   scale: 1 }
-  kubeflow-dashboard:        { charm: cs:kubeflow-dashboard-56,       scale: 1 }
-  kubeflow-profiles:         { charm: cs:kubeflow-profiles-52,        scale: 1 }
-  kubeflow-volumes:          { charm: cs:kubeflow-volumes-0,          scale: 1 }
-  mlmd:                      { charm: cs:mlmd-5,                      scale: 1 }
-  minio:                     { charm: cs:minio-55,                    scale: 1 }
-  oidc-gatekeeper:           { charm: cs:oidc-gatekeeper-54,          scale: 1 }
-  pytorch-operator:          { charm: cs:pytorch-operator-53,         scale: 1 }
+  kfp-persistence:           { charm: kfp-persistence,         channel: edge, scale: 1 }
+  kfp-schedwf:               { charm: kfp-schedwf,             channel: edge, scale: 1 }
+  kfp-ui:                    { charm: kfp-ui,                  channel: edge, scale: 1 }
+  kfp-viewer:                { charm: kfp-viewer,              channel: edge, scale: 1 }
+  kfp-viz:                   { charm: kfp-viz,                 channel: edge, scale: 1 }
+  kubeflow-dashboard:        { charm: kubeflow-dashboard,      channel: edge, scale: 1 }
+  kubeflow-profiles:         { charm: kubeflow-profiles,       channel: edge, scale: 1 }
+  kubeflow-volumes:          { charm: kubeflow-volumes,        channel: edge, scale: 1 }
```

Comment:
Please also add the kubeflow-tensorboard charms which should relate with istio-pilot